transmart-core-db/src/test/groovy/org/transmartproject/db/dataquery/highdim/rnaseq/transcript/DeTranscriptAnnotationSpec.groovy

```diff
@@ -0,0 +1,22 @@
+package org.transmartproject.db.dataquery.highdim.rnaseq.transcript
+
+import grails.test.mixin.TestFor
+import spock.lang.Specification
+
+/**
+ * See the API for {@link grails.test.mixin.domain.DomainClassUnitTestMixin} for usage instructions
+ */
+@TestFor(DeRnaseqTranscriptAnnot)
+class DeTranscriptAnnotationSpec extends Specification {
```

Comment:
These domain classes are being tested in the end to end retrieval tests, so this empty test class can be safely removed.